import numpy as np

<<<<<<< HEAD
np.Inf = np.Inf  # E: Cannot assign to final
np.ALLOW_THREADS = np.ALLOW_THREADS  # E: Cannot assign to final
np.little_endian = np.little_endian  # E: Cannot assign to final
np.UFUNC_PYVALS_NAME = "bob"  # E: Incompatible types
np.CLIP = 2  # E: Incompatible types
=======
np.little_endian = np.little_endian  # E: Cannot assign to final
>>>>>>> Final updates for Fresh Finder project
