import numpy as np

<<<<<<< HEAD
# Technically this works, but probably shouldn't. See
#
# https://github.com/numpy/numpy/issues/16366
#
np.maximum_sctype(1)  # E: No overload variant

np.issubsctype(1, np.int64)  # E: incompatible type
=======
np.isdtype(1, np.int64)  # E: incompatible type
>>>>>>> Final updates for Fresh Finder project

np.issubdtype(1, np.int64)  # E: incompatible type
